branch_training/Andrej

```diff
@@ -0,0 +1 @@
+Hello Mars
```

Comment:
in my books a haiku has 3 lines. Come on Helmut. You can improve this. and did you create your own file ? or modify somebody's else's ?